go.mod

```diff
@@ -111,14 +114,20 @@ require (
 	go.etcd.io/bbolt v1.3.6 // indirect
 	golang.org/x/crypto v0.0.0-20210915214749-c084706c2272 // indirect
 	golang.org/x/net v0.0.0-20211005001312-d4b1ae081e3b // indirect
-	golang.org/x/sys v0.0.0-20211205182925-97ca703d548d // indirect
+	golang.org/x/sys v0.0.0-20220114195835-da31bd327af9 // indirect
 	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1 // indirect
 	golang.org/x/text v0.3.7 // indirect
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
 	gopkg.in/ini.v1 v1.66.2 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 	nhooyr.io/websocket v1.8.6 // indirect
 )
 
+require (
+	github.com/dgraph-io/badger/v2 v2.2007.3 // indirect
+	github.com/prometheus/client_golang v1.12.0
+)
```

Comment:
Don't really have an opinion on this, but seems odd that we have another require section.

Reply:
Hmmm... I think this came from `go mod tidy`. I didn't play with this file myself. just using the go tooling

Reply:
I can merge them together gladly